examples/cloud-operations/network-dashboard/main.tf

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id_list = toset(var.monitored_projects_list)
+  projects        = join(",", local.project_id_list)
+
+  limit_subnets_list       = tolist(var.limit_subnets)
+  limit_subnets            = join(",", local.limit_subnets_list)
+  limit_instances_list     = tolist(var.limit_instances)
+  limit_instances          = join(",", local.limit_instances_list)
+  limit_instances_ppg_list = tolist(var.limit_instances_ppg)
+  limit_instances_ppg      = join(",", local.limit_instances_ppg_list)
+  limit_vpc_peer_list      = tolist(var.limit_vpc_peer)
+  limit_vpc_peer           = join(",", local.limit_vpc_peer_list)
+  limit_l4_list            = tolist(var.limit_l4)
+  limit_l4                 = join(",", local.limit_l4_list)
+  limit_l7_list            = tolist(var.limit_l7)
+  limit_l7                 = join(",", local.limit_l7_list)
+  limit_l4_ppg_list        = tolist(var.limit_l4_ppg)
+  limit_l4_ppg             = join(",", local.limit_l4_ppg_list)
+  limit_l7_ppg_list        = tolist(var.limit_l7_ppg)
+  limit_l7_ppg             = join(",", local.limit_l7_ppg_list)
+}
+
+################################################
+# Monitoring project creation                  #
+################################################
+
+module "project-monitoring" {
+  source          = "github.com/GoogleCloudPlatform/cloud-foundation-fabric///modules/project"
```

Comment:
I'd ping all these modules to specific versions too.

Reply:
And I think you have an extra slash extra slash before "modules/". This one in particular should be ""github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/project""

Reply:
Done in my 2nd commit, thanks.